server/polar/order/schemas.py

```diff
@@ -68,6 +79,14 @@ class OrderBase(TimestampedSchema, IDSchema):
     )
     billing_address: Address | None
 
+    @field_serializer("billing_reason")
+    def serialize_billing_reason(
```

Comment:
Nice! Could we add a simple test for that? A simple schema test in `tests/order/test_schemas.py` can do.